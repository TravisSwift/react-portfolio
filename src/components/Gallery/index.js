import React from 'react';
import PhotoList from '../PhotoList';
import { capitalizeFirstLetter } from '../../utils/helpers';

function Gallery({  }) {

  return (
    <section>
      <PhotoList category={'gallery'} />
    </section>
  );
}

export default Gallery;