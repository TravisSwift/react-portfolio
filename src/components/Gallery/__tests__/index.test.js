import React from 'react'
import { render, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Gallery from '..'

const portfolio = { name: "portfolio", description: "images of apps" };

afterEach(cleanup)

describe('Gallery is rendering', () => {

  it('renders', () => {
    render(<Gallery currentCategory={portfolio} />);
  });
})

it('matches snapshot', () => {
    const { asFragment } = render(<Gallery currentCategory={portfolio} />)
    expect(asFragment()).toMatchSnapshot()
  })