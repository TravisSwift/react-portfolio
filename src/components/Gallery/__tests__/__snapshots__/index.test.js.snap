// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <section>
    <h1
      data-testid="h1tag"
    >
      Portfolio
    </h1>
    <p>
      images of apps
    </p>
    <div>
      <div
        class="flex-row"
      />
    </div>
  </section>
</DocumentFragment>
`;
